schemas/constructs/v1alpha1/relationshipEvaluation.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$id": "https://schemas.meshery.io/evaluationResponse.json",
```

Comment:
The intermixing of OpenAPI 3.1.0 above and JSON Schema draft 7 here confuses me a bit.

Reply:
yeah , makes sense to me to have it as openapi schema.

Reply:
we need stricter guidelines to where which types of schema's belong

Reply:
I propose to migrate all schemas to be in openapi format.